### DISCUSSION TITLE
Account Suspended

### DISCUSSION THREAD

**Discussion by simanto848** [community/community#58191] [Other Feature Feedback, Questions, & Ideas] [answered]
How to unsuspended account?

**Comment by the-officialjosh** [comment]
If your GitHub account has been suspended, you will need to follow certain steps to resolve the issue and have your account reinstated. Here's what you can do:

1. Review the suspension email: GitHub usually sends an email explaining the reason for the suspension. Read it carefully to understand why your account was suspended and what actions you need to take to resolve the situation.

2. Contact GitHub support: Reply to the suspension email or reach out to GitHub support directly through their contact page. Clearly state your situation, provide any relevant information or context, and express your desire to have your account reinstated. Be polite and concise in your communication.

3. Rectify the issue: If the suspension was due to a specific policy violation or misuse of GitHub's services, you may need to rectify the issue before your account can be reinstated. This might involve removing certain content, addressing violations, or correcting any mistakes you made.

4. Cooperate with GitHub: If GitHub requires additional information or evidence from you, promptly provide them with the requested details. Be responsive and cooperative during the process, as it can help expedite the resolution.

5. Wait for a response: GitHub support will review your case and respond to your request. The response time may vary depending on the complexity of the situation and their workload. It's important to be patient during this period.

6. Follow any additional instructions: If GitHub provides instructions or guidance on specific actions you need to take to reinstate your account, make sure to follow them diligently. This might include agreeing to certain terms, acknowledging mistakes, or taking further actions to resolve the situation.

7. Maintain a positive approach: Throughout the process, remain professional, cooperative, and respectful in your interactions with GitHub support. A positive attitude can go a long way in resolving the issue and having your account unsuspended.

Remember that these steps are general guidelines, and the specific process to unsuspend your GitHub account may vary depending on the reason for the suspension. It's essential to carefully read the communication from GitHub and follow their instructions accordingly.

**Comment by queenofcorgis** [comment, marked as answer]
Hi @simanto848 ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :) Given that the community can't assist with this, I'm going to close and lock this post - good luck!
